samples/Http2SampleApp/scripts/build-docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+dotnet publish --framework netcoreapp2.0 "$DIR/../Http2SampleApp.csproj"
```

Comment:
Nit: Is `DIR` necessary since you've already `cd`d into it?

Reply:
Nope, the above code doesn't `cd` in the main context, it creates a subshell with `$()`